src/graphology/tests/mutation.js

```diff
@@ -388,6 +388,13 @@ export default function mutation(Graph, checkers) {
         }, usage());
       },
 
+       'it should be able to merge undirected edges in both directions': function () {
+        const graph = new Graph();
+        graph.mergeUndirectedEdgeWithKey('J<->M', 'John', 'Martha');
```

Comment:
You should wrap those calls in [`assert.doesNotThrow`](https://nodejs.org/api/assert.html#assertdoesnotthrowfn-error-message), using a usage error.

Reply:
Ok - I imported UsageGraphError from ../src/errors.

Reply:
You don't need to. You just need to call the `usage` function provided to the test for dependency injection purposes as does the test just before yours (line 388).

Reply:
Gotcha - didn't see that! Committed.